homeassistant/components/ituran/quality_scale.yaml

```
@@ -0,0 +1,89 @@
+rules:
+  # Bronze
+  config-flow: done
+  test-before-configure: done
+  unique-config-entry: done
+  config-flow-test-coverage: done
+  runtime-data: done
+  test-before-setup: done
+  appropriate-polling: done
+  entity-unique-id: done
+  has-entity-name: done
+  entity-event-setup:
+    status: exempt
+    comment: |
+      Entities of this integration does not explicitly subscribe to events.
+  dependency-transparency: done
+  action-setup:
+    status: exempt
+    comment: |
+      This integration does not provide additional actions.
+  common-modules: done
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: done
+  docs-actions:
+    status: exempt
+    comment: |
+      This integration does not provide additional actions.
+  brands: done
+  # Silver
+  config-entry-unloading: done
+  log-when-unavailable: done
+  entity-unavailable: done
+  action-exceptions:
+    status: exempt
+    comment: |
+      This integration does not provide additional actions.
+  reauthentication-flow: done
+  parallel-updates:
+    status: exempt
+    comment: |
+      Read only platforms and coordinator.
+  test-coverage: done
+  integration-owner: done
+  docs-installation-parameters: done
+  docs-configuration-parameters:
+    status: exempt
+    comment: |
+      No options flow.
+  # Gold
+  entity-translations: done
+  entity-device-class:
+    status: exempt
+    comment: |
+      Only device_tracker platform.
+  devices: done
+  entity-category: todo
+  entity-disabled-by-default: todo
+  discovery:
+    status: exempt
+    comment: |
+      This integration cannot be discovered, it is a connecting to a service
+      provider, which uses the users credentials to get the data.
+  stale-devices: todo
+  diagnostics: todo
+  exception-translations: done
+  icon-translations: done
+  reconfiguration-flow: todo
+  dynamic-devices: done
+  discovery-update-info:
+    status: exempt
+    comment: |
+      This integration cannot be discovered, it is a connecting to a service
+      provider, which uses the users credentials to get the data.
+  repair-issues:
+    status: exempt
+    comment: |
+      No repairs/issues.
+  docs-use-cases: todo
+  docs-supported-devices: done
```

```suggestion
  docs-supported-devices: todo
```
at least for me it's not clear from the docs what's supported

---

Basically, from the documentation:
>You must have an Ituran account for use with the Ituran APP

If you had their device installed in your car, it'll be supported by this integration as it only communicates with their web services and doesn't really care about your specific car model. I'm hoping that would be clear to anyone who make use of their product but, if you think it can be clearer, please let me know how.

---

they have probably more devices? At least I don't understand from their website what I would need to buy to use this integration

---

I'm sure they have multiple devices and it depends on what the insurance companies require but in top of their devices you need to pay an extra premium for their mobile app service. Assuming you have that service, and I'm assuming they would only (up)sell you that service if the hardware supports it, this integration should work.
I, basically, reversed engineered what their app does to access the same information.

---

then I'd add something along the lines "all devices that work with the app should work"

---

Done